analyzers/src/SonarAnalyzer.Common/Rules/Hotspots/HardcodedIpAddressBase.cs

```diff
@@ -46,6 +46,20 @@ public abstract class HardcodedIpAddressBase<TSyntaxKind, TLiteralExpression> :
             "ASSEMBLY",
         };
 
+        // https://datatracker.ietf.org/doc/html/rfc5737
+        private readonly byte[][] interNetworkDocumentationRanges =
+        {
+            new byte[] { 192, 0, 2 },    // TEST-NET-1 192.0.2.0/24
+            new byte[] { 198, 51, 100 }, // TEST-NET-2 198.51.100.0/24
+            new byte[] { 203, 0, 113 },  // TEST-NET-3 203.0.113.0/24
```

Comment:
Are the comments really needed? I find it obvious

Reply:
It refers to the ranges in the RFC. You can search for "TEST-NET-1" and you will find the definition there. It is helpful but not really necessary.